clients/src/main/resources/common/message/MetadataRequest.json

```diff
@@ -31,9 +31,11 @@
     // Starting in version 8, authorized operations can be requested for cluster and topic resource.
     //
     // Version 9 is the first flexible version.
+    // Version 10 add topicId
     { "name": "Topics", "type": "[]MetadataRequestTopic", "versions": "0+", "nullableVersions": "1+",
       "about": "The topics to fetch metadata for.", "fields": [
-      { "name": "Name", "type": "string", "versions": "0+", "entityType": "topicName",
+      { "name": "TopicId", "type": "uuid", "versions": "10+", "ignorable": true, "about": "The topic id." },
+      { "name": "Name", "type": "string", "versions": "0+", "entityType": "topicName", "ignorable": true,
```

Comment:
As discussed in the mailing thread and the KIP writeup, this should be "nullable"

Reply:
I tried to use nullable but got an Exceptioin with `Unrecognized field "nullable"` message.